package/utils/dtc/Makefile

```diff
@@ -102,6 +103,11 @@ define Build/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/* $(1)/usr/lib
 endef
 
+define Host/Install
+	$(INSTALL_DIR) $(STAGING_DIR_HOSTPKG)/bin/
+	$(CP) $(MESON_HOST_BUILD_DIR)/dtc $(STAGING_DIR_HOST)/bin/
+endef
+$(eval $(call HostBuild))
```

Comment:
for TF-A we just use the dtc binary from the linux kernel, see https://github.com/openwrt/openwrt/blob/main/include/trusted-firmware-a.mk#L77.

Where do you need this dtc binary?

Reply:
I needed dtc binary for optee-os package.
And I forgot to include kernel.mk in arm-trusted-firmware-stm32.

Yes I can use dtc binary from linux kernel for optee and I added missing kernel.mk in arm-trusted-firmware-stm32 makefile, so this is not needed anymore.